docs/references/HCPEtcdBackup/HCPEtcdBackup-implementation.md

```
@@ -0,0 +1,542 @@
+  - resticrepositories.velero.io
+```
+
+3. Verify the restored HostedCluster has `restoreSnapshotURL` injected:
+
+```bash
+oc get hostedcluster <name> -n <hosted-cluster-namespace> \
+  -o jsonpath='{.spec.etcd.managed.storage.restoreSnapshotURL}'
+```
+
+The output should be an array containing a **pre-signed HTTPS URL**, e.g.:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)